.travis.yml

```diff
@@ -19,6 +19,8 @@ matrix:
 
 
 before_install:
+    # See https://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-a-GUI :
+    - "/sbin/start-stop-daemon --start --quiet --pidfile /tmp/custom_xvfb_99.pid --make-pidfile --background --exec /usr/bin/Xvfb -- :99 -ac -screen 0 1280x1024x16"
```

Comment:
is this for `visbrain`?

For matplotlib there is a simpler solution ...

Reply:
Well, not only for visbrain. See for example for [mne](https://github.com/mne-tools/mne-python/blob/master/.travis.yml#L52)

Reply:
yes that's for `mayavi` but I would suggest adding it when needed. It shouldn't be necessary for PSD plots

Reply:
Indeed, tests are still working without this line.

Reply:
Now the only thing that need to be add to travis is : 
```
cache:
  directories:
  - $HOME/mne_data/
```
?